src/params/types.ts

```diff
@@ -345,7 +345,13 @@ export class SecretParam {
 
   /** @internal */
   runtimeValue(): string {
-    return process.env[this.name] || "";
+    const val = process.env[this.name];
+    if (!val) {
```

Comment:
couldn't someone deliberately set a secret to the empty string, which is falsy?

Reply:
Changed from a falsy check to `val === undefined`